service-deployment/src/grafana/pai-clusterview-dashboard.json.template

```diff
@@ -42,13 +43,14 @@
             "thresholdLabels": false,
             "thresholdMarkers": true
           },
+          "height": "312px",
           "id": 9,
           "interval": null,
           "links": [
             {
               "dashboard": "PAI Cluster Node List",
               "type": "absolute",
-              "url": "{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_url'] }}:{{ clusterconfig['clusterinfo']['grafanainfo']['grafana_port'] }}/dashboard/db/pai_clusterview"
+              "url": "http://10.151.40.179:3000/dashboard/db/pai_clusterview"
```

Comment:
I guess this change is for testing purpose on devbox? Should change back to value from clsuterconfig.

Reply:
Yes, Solved in latest commit